docs/validators/key-management/multikey-nodes.md

```diff
@@ -0,0 +1,123 @@
+---
+id: multikey-nodes
+title: Multikey nodes management
+---
+
+[comment]: # (mx-abstract)
+This page contains information about how to manage multiple keys on an observing squad.
+
+[comment]: # (mx-context-auto)
+## Multikey architecture overview
+
+At the mainnet launch, and up until the release candidate RC/v1.6.0 becomes mainstream, each node could have managed only
+one key. So the relationship between the nodes and keys is `1:1`, the node "following" the shard where the managed key is assigned.
+The main idea behind the multikey feature is that a set of keys can be managed by an observing squad set. The same set of keys
+should be provided for all observing squad nodes, so the nodes can **pick** any key from that set and use it to propose or sign
+blocks whenever the consensus group in a round contains at least one managed key. The observing node that runs in multikey mode
+is able to use the provided keys in any combination possible. So one node can use one key to propose a block and one or more other
+keys to already sign on that proposed block automatically to improve performance.
+So, with the multikey feature, the relationship now becomes `n:[num_shards]`, providing that `n` is the number of keys 
+managed by an entity.
+
+:::important
+This feature is purely optional. Normal `1:1` relationship between the keys and the nodes is still supported. The multikey
+mode should optimize the costs when running a set of keys (check [Economics running multikey nodes](#economics-running-multikey-nodes) section)
+:::
+
```

Comment:
you could include a diagram. I'll attach a drawio diagram that I've created
[multikey-diagram.zip](https://github.com/multiversx/mx-docs/files/11722196/multikey-diagram.zip)

Reply:
done, thanks